### DISCUSSION TITLE
Welcome to Panasonic Comfort Cloud Home Assistant Discussions!

### DISCUSSION THREAD

**Discussion by sockless-coding** [sockless-coding/panasonic_cc#229] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by qsilver84** [comment]
Hi,

I install this addon but i got the message "No devices" after login process.

Thanks 
Regards

**Reply by qsilver84** [reply]
![image](https://github.com/user-attachments/assets/c0dbe882-4d93-4934-b1b0-f14909183cec)

**Reply by sockless-coding** [reply]
@qsilver84 What heat pump do you have?
Also, could you provide some logs please see https://github.com/sockless-coding/panasonic_cc/issues/256#issuecomment-2368204021

**Comment by notio-ca** [comment]
Thanks for this great integration! Works great!

Is there a way to read if the outside heat pump is in Defrost Cycle ? I don't see any sensors.
Light should flash on the inside unit when this is the case.
Unit are not heating for 15 to 20 minutes during a defrost cycle, I would like to trigger another heating system in this period.

Thanks for your help!

**Comment by sockless-coding** [comment]
@notio-ca unfortunately, this information is not provided by the API. And the power utilization updates are too slow to be used to detect it (refreshes every 15 min).

**Comment by Bobosko** [comment]
Hello, when adding the integration to HA, I get an error message - Unexpected error while creating device.
Can I ask for help?
Thank you

**Reply by notio-ca** [reply]
Integration is broken at this time, Panasonic changed something on their cloud

**Comment by notio-ca** [comment]
@sockless-coding 
Special thanks to Jimmy Everling for this great feature below.
I have a 4 heads system thought out the house, this integration is gold to me.
Your work is appreciated, just wanted to let you know
![thermo](https://github.com/user-attachments/assets/82eaa421-e917-4cd4-8240-4183057445fb)

**Comment by Jez2730** [comment]
Hello, Thank you very much for developing that ! I have an issue when starting the integration, I get the following: Panasonic Comfort Cloud
Unexpected error creating device.

I did activate the 2FA authentication by sms -- Maybe shall I reset Home assistant afterward ? 
Thank you :)

**Comment by juzel69** [comment]
Hello. I installed the integration. But i can't login. It always shows 'Unknown error occured'. The Login data (E-Mail, Password) works with the Comofrt Cloud App. What could be my problem? Thanks.
